.github/workflows/test-exports.yml

```diff
@@ -134,7 +134,7 @@ jobs:
       - name: Install dependencies
         run: yarn install --immutable
       - name: Build
-        run: yarn workspace langchain-core build && yarn workspace langchain build
+        run: yarn workspace langchain-core build && yarn workspace @langchain/anthropic build && yarn workspace langchain build
```

Comment:
should we add this command to package json?

Reply:
Yes probably, will do when I do OpenAI